src/components/hero.tsx

```diff
@@ -9,37 +9,46 @@ type HeroProps = {
 };
 
 export function Hero({ allEpisodes }: HeroProps) {
+  if (!allEpisodes || allEpisodes.length < 3) {
+    return null;
+  }
+
   return (
     <div className="m-auto -mb-32 h-screen w-3/4 md:-mb-48">
       <div className="h-2/3 flex-col gap-4 overflow-hidden md:flex md:flex-row">
         <HeroCard
           heroLink={`/podcasts/${allEpisodes[0].guid}`}
           imageUrl={allEpisodes[0].itunes.image}
           heroTitle={allEpisodes[0].title}
-          size={'lg'}
+          size="lg"
         />
         <div className="flex flex-1 flex-col gap-4 overflow-hidden">
           <HeroCard
             heroLink={`/podcasts/${allEpisodes[1].guid}`}
             imageUrl={allEpisodes[1].itunes.image}
             heroTitle={allEpisodes[1].title}
-            size={'md'}
+            size="md"
           />
           <HeroCard
             heroLink={`/podcasts/${allEpisodes[2].guid}`}
             imageUrl={allEpisodes[2].itunes.image}
             heroTitle={allEpisodes[2].title}
-            size={'md'}
+            size="md"
           />
         </div>
       </div>
-      <div className=" mt-12 flex justify-end md:mt-8">
-        <Link to={'/podcasts'}>
-          <Button variant="outline" size="s">
+      <div className="mt-12 flex justify-end md:mt-8">
+        <Link to="/podcasts">
+          <Button
```

Comment:
do not dupplicate button style 

create a new variant if you want or update the others ones 
https://github.com/flexbox/artaufeminin/blob/master/src/components/button.tsx#L9

you can have an overview of all components running

```
yarn storybook
```